pom.xml

```diff
@@ -98,15 +108,37 @@
                     <!-- <useIncrementalCompilation>false</useIncrementalCompilation> -->
 
                     <annotationProcessorPaths combine.children="append">
+                        <path>
+                            <!-- must precede micronaut-inject-java -->
```

Comment:
why ? What's the problem ?

Reply:
This order guarantees that the Lombok processor will run before the Micronaut processor.
The issue is documented among the micronaut-core issues ([here](https://github.com/micronaut-projects/micronaut-core/issues/218)).